deploy/gen-messages-exceptions.py

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env python
```

Comment:
this shebang is missing from other files.. and should probably rather be python3.. well you are calling it explicitly with ${PYTHON} so it does not really matter for the auto generation. But for testing and developing it may be nice to be able to quickly call a single generator. In the end it depends on if you want these to be chmod-executable or not. they should be all alike though.

Reply:
I generally avoid putting the shebang line, I think this one is from the original file I modified this from. Ideally a human shouldn't be calling this script unless they're updating the script, it should be called by `bootstrap` or from CMake directly after the merge. I'll remove it from this so it's clear you shouldn't be `./` calling these files.